### PR TITLE
various content tweaks

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -17,4 +17,6 @@ We will accomplish our mission by:
 * being design-centric, agile, open, and data-driven
 * deploying tools and services early and often
 
+We know, there is a lot of information on this site, which can be intimidating. Federal hiring processes are more complex than most other places, so we want to make sure you have all the information you need. We're happy you're interested in joining the team!
```
Comment (afeld) [inline pages/index.md:20]: My thinking here is that this site is great in that it includes a _ton_ of information, but I worry that it's scaring away potential applicants. Tried to make it sound friendly and inviting, and to allay (or confirm?) their fears that "government application processes are scary".

We might also consider distinguishing pages that are "required" reading before they move to the application, vs. those that are there simply there for reference (or at least linked to in context on the application form). Could be as simple as linking to them from a final sentence in this paragraph. My thinking is that they only really _need_ to read the following pages:
- Who we are hiring (maybe?)
- Roles and teams
- How to apply

Thoughts?

File: pages/roles-and-teams.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Roles and teams
 permalink: /roles-and-teams/
 ---
-Although everyone is officially hired into the GSA as an Innovation Specialist, our Innovation Specialists fill a wide range of roles on a number of different teams across 18F. Below are a few of the roles that Innovation Specialists may fill at 18F. This is not intended to be a comprehensive list but for those that are interested in joining the team, hopefully something on this list excites you!
+Although everyone is officially hired into the GSA as an Innovation Specialist, our Innovation Specialists fill a wide range of roles on a number of different teams across 18F. Below are a few of the roles that Innovation Specialists may fill at 18F. This is not intended to be a comprehensive list, but for those that are interested in joining the team, hopefully something on this list excites you!
```
Comment (fureigh) [inline pages/roles-and-teams.md:5]: Good catch, thanks.

File: pages/who-we-are-hiring.md [hunk 3]
```diff
@@ -7,8 +7,9 @@ permalink: /who-we-are-hiring/
...
 
 ## Where we are hiring
 
-Everywhere in the United States! Most of our team is distributed across the country in places like Chicago, New York, Raleigh, Tucson, Austin, Dayton, Philadelphia, San Diego, Seattle, and Portland. [Read more about how our work culture supports distributed teams.](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/) In some cases we'll ask you to work out of our offices in San Francisco, Chicago or D.C.
+Everywhere in the United States! Most of our team is distributed across the country in places like Chicago, New York, Raleigh, Tucson, Austin, Dayton, Philadelphia, San Diego, Seattle, and Portland. [Read more about how our work culture supports distributed teams.](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/) In some cases we'll ask you to work out of our offices in San Francisco, Chicago, or D.C. Unfortunately, we can't provide relocation assistance.
```
Comment (fureigh) [inline pages/who-we-are-hiring.md:15]: Excellent point to include.

File: pages/index.md (hunk 1 above)
Comment (fureigh) [inline pages/index.md:20]: Good call. I like the idea of more clearly communicating the pages' hierarchy of importance (i.e., which are "required" and which are simply there for reference). If there are pages that must be read before applying, though, I'd like us to ensure that site visitors encounter them en route to the application, as we can't guarantee that they'll land on this page first.

I share your assessment of those three pages (including the "maybe" about the "Who we are hiring" page.) @rooneywp, what do you think?

**Comment by fureigh** [comment]
@afeld Thanks for these! I noted a question about a possible enhancement above but will merge this PR after resolving conflicts.

**Pull request merged**